satpy/etc/readers/eps_sterna_l1b_nc.yaml

```diff
@@ -0,0 +1,532 @@
+reader:
```

Comment:
Hmm. Shouldn't there be only one reader named `mws_l1b_nc` and another for the level 1c data? The L1b reader for the MWS instrument should be the same for AWS-PFS and the future Sterna constellation.

Reply:
No, unfortunately the formats are not the same! ESAs format is different. So one format for the AWS PFM, and another for the EPS-Sterna. So, we need two readers for the l1b.